e2e/tests/components/select/standard/select.e2e-spec.ts

```diff
@@ -591,7 +597,8 @@ describe('Select Tests', () => {
         await page.clickOnDropdown(true);
 
         // Initial state: recent option list is not shown
-        // expect(await imageCompare('select-open-multi')).toEqual(0);
+        browser.wait(EC.invisibilityOf(page.checkRecentOptionsList()), 5000);
```

Comment:
This will also pass if the dropdown isn't present, so it doesn't really help us.